app/models/street/StreetEdgePriorityTable.scala

```diff
@@ -71,16 +69,6 @@ object StreetEdgePriorityTable {
     streetEdgePriorities.filter{ edg => edg.streetEdgeId === streetEdgeId}.map(_.priority).list.head
   }
 
-  def getAllStreetEdgeInRegionPriority(regionId: Int): List[StreetEdgePriority] = db.withTransaction { implicit session =>
```

Comment:
This was also code for the ajax call to get the priorities of the streets, which has been replaced by including the priority in the NewTask objects.